DSCResources/MSFT_xNetBIOS/MSFT_xNetBIOS.psm1

```diff
@@ -0,0 +1,112 @@
+Add-Type -TypeDefinition @'

```

Comment:
You'll want to wrap this in an if statement like [xDSCResourceDesigner](https://github.com/PowerShell/xDSCResourceDesigner/blob/dev/xDSCResourceDesigner.psm1#L100).

This prevents errors like this:

````powershell
Add-Type : Cannot add type. The type name 'NetBiosSetting' already exists.
At C:\GitHub\tysonjhayes\xNetworking\DSCResources\MSFT_xNetBIOS\MSFT_xNetBIOS.psm1:1 char:1
+ Add-Type -TypeDefinition @'
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (NetBiosSetting:String) [Add-Type], Exception
    + FullyQualifiedErrorId : TYPE_ALREADY_EXISTS,Microsoft.PowerShell.Commands.AddTypeCommand
````

Which happened to me as I was testing something then ran the unit tests.